package.json

```diff
@@ -29,38 +29,35 @@
   "homepage": "https://github.com/warehouseai/workers-factory#readme",
   "dependencies": {
     "@wrhs/extract-config": "^1.0.0",
-    "async": "~2.1.4",
-    "babel-core": "~6.21.0",
-    "babel-preset-es2015": "~6.18.0",
-    "babel-preset-react": "~6.16.0",
-    "bl": "~1.2.0",
-    "browserify": "~13.3.0",
-    "clean-css": "~3.4.23",
+    "async": "~3.1.0",
```

Comment:
Since you're touching them... Do we want to start using `^` here or stick with `~`?